### ISSUE TITLE
DUP feedback research and report out

### ISSUE THREAD

**Issue by kgorana** [bcgov/parks-data-register#526]
## Description
This ticket is to highlight and share any noteworthy changes received in user feedback (ParkInfo DUP emails) based on the recent steps taken to improve users' experience:
- noteworthy user frustrations
- user recommendations
- changes in number of questions after publishing of FAQ page

The study will be scoped to the most critical months i.e. **June, July, and August months** from **2023 vs 2024**.

## Acceptance Criteria
As an interested party,
I want to see if there have been any improvements to the DUP program
given the modifications made intended to reflect better user experience.

## Development Checklist
- [ ] ...
- [ ] ...
- [ ] ...

### Blocked by
- [ ] Access to the ParkInfo DUP folder mailbox
- [ ] Download of the email data
- [ ] Clean-up of the email data

## Definition of Ready
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

### In progress
- [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
- [ ] Report out meets accessibility requirements
- [ ] Unit tests are written
- [ ] Work is traceable in GitHub
- [ ] PR linked to ticket number
- [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment

### SME review
- [ ] Study and Report is peer reviewed and has passed SME expectations and/or queries

### PO Review
- [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
- [ ] Reviewed and approved by Product Owner

## Notes:
- https://reserve.bcparks.ca/dayuse/faq
-

**Comment by LindsayMacfarlane** [comment]
@Dianadec Do we still need?

**Comment by LindsayMacfarlane** [comment]
Confirmed with Diana and Jess that this ticket is no longer needed. Closing.
